src/components/Cart/Cart.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect } from "react";
+import { CartProducts } from "../../context/CartProvider";
+
+export function Cart() {
+	const { products, removeFromCart, total } =
+		CartProducts();
+
+	useEffect(() => {});
+
+	let totalDisplayscont = total - 165;
+	return (
+		<>
+			<div className="user-main-container flex-space-evenly-start">
+				<main className="product-container border-radius-normal box-shadow col-7">
+					<p className="wishlist-text padding-normal">
+						My Cart({products.length})
+					</p>
+					{products.map((product) => {
+						return (
+							<div
+								className="wishlist-card-container"
+								key={product.id}
+							>
+								<div className="card-container-horizontal flex">
+									<section className="cart-product flex-column-center">
+										<section className="flex-row-start">
+											<img
+												className="imgcard cart-img"
+												src={product.img}
+												alt=""
```

Comment:
Suggestion : Give a value for alt attribute

Reply:
Okay, I'll do the suggested changes Thank you Balaji for PR Review